scripts/kconfig/kconfigfunctions.py

```diff
@@ -335,11 +335,47 @@ def dt_compat_enabled(kconf, _, compat):
     return "n"
 
 
+def dt_compat_get_string(kconf, _, compat, inst=0, ret_def='', prop='label'):
+    """
+    This function takes a 'compat' and an 'inst'ance to search for an EDT node that
+    matchs. If it finds an EDT node, it will look to see if that node has a name
+    property by the name of 'prop'. If the 'prop' exists it will return the string
+    representation of that 'prop' otherwise return ret_def.
+    """
+
+    ret_def_trimed = ret_def.strip()
```

Comment:
*trimmed

Reply:
Fixed with variables **\<parameter\>_s**. I noted that if user break lines at _Kconfig_ will be necessary execute **strip()** to avoid miss behaviors on all strings. This must be applied on all existing methods when applicable.